src/buildMathML.js

```diff
@@ -167,6 +168,20 @@ groupTypes.color = function(group, options) {
 };
 
 groupTypes.supsub = function(group, options) {
+    // Is the inner group a relevant horizonal brace?
+    let isBrace = false;
+    let isOver;
+    let isSup;
+    if (group.value.base) {
+        if (group.value.base.value.type === "horizBrace") {
+            isSup = (group.value.sup ? true : false);
```

Comment:
Yes. Say that an author writes `\overbrace{...}^{note}`.  Then both `group.value.sup` and `group.value.base.value.isOver`  will be true, `.isSup` will be true, and control over `group.value.sup` will pass to `groupTypes.horizBrace`. 

If the author writes `\overbrace{...}_{note}` (don't ask me why), then `group.value.sup` will be false and control over `group.value.sub` will pass to `groupTypes.supsub` to be written as a normal subscript.